asm/banks/df.asm

```diff
@@ -0,0 +1,284 @@
+hirom
+
+; DF Bank
+
+; ########################################################################
+; DFB600-DFB9FF: Colosseum Data (256 items, 4 bytes each) 
+;
+; -----------------------------------------------------------------------------
+; Description
+; -----------------------------------------------------------------------------
+; This hack modifies the colosseum data of each betting item to store
+; its color palette in byte $03 (used for "show/hide reward" info).
+;
+; vanilla: $00 = show reward / $FF = hide reward
+;     now: $00 = hide reward / $XX = color palette
+; -----------------------------------------------------------------------------
+
+!open  = $20     ; white text color
+!closed = $34     ; yellow text color
```

Comment:
It appears that the "yellow" palette is not defined yet. There must be another `asm` that this patch depends on...